use_case_examples/dataframe/encrypted_pandas.ipynb

```diff
@@ -0,0 +1,321 @@
+    ")\n",
+    "\n",
+    "pandas.set_option(\"display.max_columns\", None)\n",
+    "pandas.set_option(\"display.width\", 1000)\n",
+    "numpy.random.seed(0)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Clients"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Path directory for client and server files\n",
+    "INPUTS_OUTPUTS_DIR = Path(\"inputs_outputs\")\n",
+    "DATA_DIR = Path(\"data\")\n",
+    "KEYS_DIR = Path(\"keys\")\n",
+    "\n",
+    "if INPUTS_OUTPUTS_DIR.is_dir():\n",
+    "    shutil.rmtree(INPUTS_OUTPUTS_DIR)\n",
+    "if KEYS_DIR.is_dir():\n",
+    "    shutil.rmtree(KEYS_DIR)\n",
+    "\n",
+    "INPUTS_OUTPUTS_DIR.mkdir(exist_ok=True)\n",
+    "KEYS_DIR.mkdir(exist_ok=True)\n",
+    "\n",
+    "# Pandas kwargs\n",
+    "HOW = \"left\"\n",
+    "ON = \"id\"\n",
+    "\n",
+    "# TODO: remove this if possible\n",
+    "OPERATOR = \"left_right_join\""
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "   id  feat_left_1  feat_left_2  feat_left_3\n",
+      "0   1           13            3            8\n",
+      "1   2            6            5            8\n",
+      "2   3            1            8            9\n",
+      "3   4            4            7            2\n",
+      "4   5           12            9            6\n",
+      "5   6            4            9           10\n",
+      "6   7            8           13           14\n",
+      "7   8           10           11            9\n",
+      "8   9            4            2           10\n",
+      "9  10            6            7            5\n"
+     ]
+    }
+   ],
+   "source": [
+    "df_left = pandas.read_csv(DATA_DIR / \"df_left.csv\", index_col=0)\n",
+    "df_right = pandas.read_csv(DATA_DIR / \"df_right.csv\", index_col=0)\n",
+    "\n",
+    "print(df_left)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "   id  feat_right_1  feat_right_2  feat_right_3  feat_right_4  feat_right_5\n",
+      "0   4             4             6             4             4             4\n",
+      "1   5             1             1             9            14             8\n",
+      "2   6             4             3             2             4             1\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(df_right)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "client_1, evaluation_keys_1 = get_client_and_eval_keys(OPERATOR, keys_path=KEYS_DIR / \"keys\")\n",
+    "client_2, evaluation_keys_2 = get_client_and_eval_keys(OPERATOR, keys_path=KEYS_DIR / \"keys\")"
```

Comment:
Maybe a comment about what's going on here. Currently client share the key. But could be based on a public key + TD in the future.